packages/ui/src/components/Icons/LoadingIcon.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { SVGProps } from 'react';
+
+import Loading from '@/assets/icons/loading.svg';
+
+const LoadingIcon = (props: SVGProps<SVGSVGElement>) => {
+  return (
+    <svg width="32" height="32" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg" {...props}>
+      <use href={`${Loading}#loading`} />
```

Comment:
What's good of nesting the loading.svg inside, comparing to directly writing the whole svg content here?

Reply:
Yeah just to keep the original SVG files together under the assets path.  I am expecting eventually no need to provide these hand written svg components.

Reply:
OK, I see your points. Interesting... Maybe we should do this in admin console as well

Reply:
k, leave it just for now